tsconfig.json

```diff
@@ -7,7 +7,7 @@
   ],
   "compilerOptions": {
     /* Basic Options */
-    "target": "es2020", /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
+    "target": "ES2020", /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017', 'ES2018', 'ES2019' or 'ESNEXT'. */
```

Comment:
I'd like to use `ESNext` so that we can always use the next version of ESCMAScript but some of the headset files had some issues with it that need to be looked into before we can do that.

Reply:
Am I misunderstanding what `target` does? https://www.typescriptlang.org/tsconfig/#target

When I read that, I come away with the impression that we need to consider the environment our code will run in and decide how far back we need to support (or how new we can get away with, though it looks like `ESNext` isn't really recommended).